tests/test_parser.py

```diff
@@ -495,7 +495,7 @@ def get_comb() -> num:
     return self.comb
     """
     c = get_contract_with_gas_estimation(constructor_advanced_code2, args=[[5,7], "dog", 8])
-    assert c.get_comb() == 5738
+    assert c.get_comb() == 5708
```

Comment:
Ugh, this is totally unhelpful without a comment. Why the change?

Reply:
It appears some parser functionality has been broken, because the expected value (from looking at the above contract) should be `5738`. It seems that getting `len()` of `bytes` is broken.

I tried the following using the code from the PR, and it resulted in `0 == 3`:

```
def test_len_bytes():
    test_contract = """
x: num

def __init__(y: bytes <= 3):
    self.x = len(y)
    
def get_len() -> num:
    return self.x
    """
    c = get_contract(test_contract, args=['bob'])
    assert c.get_len() == 3
```

Reply:
Thanks for the help finding/clarifying this mistake, I fixed it.  What went wrong before was that I did the math myself in a python console and must've made a typo.